src/e2e-test/regression-test/util/generate-ground-truths.js

```diff
@@ -16,7 +16,7 @@ const truthsExist = basePath => {
 
     const promises = [cdaPath, hl7v2Path].map(subPath => 
         new Promise(fulfill => fs.exists(subPath, fulfill)));
-    return Promise.all(promises).then(flags => flags.every(x => x));
+    return Promise.all(promises).then(flags => flags.some(x => x));
```

Comment:
What does this change mean here?

Reply:
The function is used to judge whether ground truth files have been generated. The existence of any part of ground truths will be regarded as the basis for not generating new ground truths. So `some` may be more appropriate than `every`.

Reply:
Is it possible that only parts of the ground truth are generated last time?

Reply:
Of course, It's all about concret implementations, and this branch is just pick one of them.